dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/model/GleifRelationshipInformation.kt

```diff
@@ -0,0 +1,20 @@
+package org.dataland.datalandbatchmanager.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Data class containing the relevant information from the GLEIF Relationship csv files
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class GleifRelationshipInformation(
+    @JsonProperty("Relationship.StartNode.NodeID")
```

Comment:
JsonIgnoreProperties allows that some attributes do not exist, right? are those fields then set o null?

Reply:
it allows the parsed csv files to have other columns than the ones given here. In the relationship csv file, there are many more columns which are skipped using this setting.